### PR TITLE
Added roman to integer conversion

### PR DESCRIPTION
**Pull request by Navid2zp**
Added roman to integer conversion and it's file link to DIRECTORY.md

### CODE REVIEW

File: conversions/roman_to_integer.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			holder += 2
+		} else {
+			total += romans[string(roman[holder])]
+			holder++
+		}
+	}
+	return total
+}
+
+func main() {
```
Comment (nomisrevol) [inline conversions/roman_to_integer.go:29]: Remove `main` function and create a new test file instead.

File: DIRECTORY.md [hunk 2]
```diff
@@ -6,6 +6,9 @@
   * [Rsacipher](https://github.com/TheAlgorithms/Go/blob/master/ciphers/RSAcipher.go)
   * [Xorcipher](https://github.com/TheAlgorithms/Go/blob/master/ciphers/xorCipher.go)
 
+## Conversions
```
Comment (nomisrevol) [inline DIRECTORY.md:9]: You don't need to update this file manually. Please remove this.

**Review by nomisrevol — CHANGES_REQUESTED** [review]

File: conversions/roman_to_integer.go (hunk 1 above)
Comment (Navid2zp) [reply conversions/roman_to_integer.go]: The main function is removed and test files created.

File: DIRECTORY.md (hunk 2 above)
Comment (Navid2zp) [reply DIRECTORY.md:9]: Not sure how to remove file changes. will deleting the file or just removing the added lines work?

File: conversions/conversions_case_test.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package conversions
+
+type romanToIntegerConversionTest struct {
+	input    string
+	expected int
+	name     string
+}
+
+var romanToIntegerTests = []romanToIntegerConversionTest{
```
Comment (nomisrevol) [inline conversions/conversions_case_test.go:9]: Move the testcases inside of the test file. We do not need it to separated in a standalone file.

File: conversions/conversions_test.go [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		actual := romanToInteger(test.input)
+		if actual != test.expected {
+			t.Errorf("roman to integer test %s failed", test.name,)
+		}
+	}
+}
+
+//END TESTS
+
+//BEGIN BENCHMARKS
```
Comment (nomisrevol) [inline conversions/conversions_test.go:18]: No need to have benchmark for a single implementation, please remove all unnecessary like `BEGIN TESTS` or `END TESTS` also.

**Review by nomisrevol — CHANGES_REQUESTED** [review]

File: conversions/conversions_case_test.go (hunk 3 above)
Comment (Navid2zp) [reply conversions/conversions_case_test.go:9]: Should I rename the test file to `roman_to_integer_test.go` or you want all the tests in one file like the way it is now?

**Pull request closed without merging**